app/views/layouts/_navbar.html.haml

```diff
@@ -29,7 +29,7 @@
                 %i.fa.fa-file-text
                 %span My Proposals
 
-          - if (current_user.reviewer? || current_user.organizer?) && event
+          - if (current_user.reviewer? || current_user.organizer?) && event && event_staff?
```

Comment:
No because this is checking if the person is specifically staff for the event. A person can have the role of reviewer but not have been invited to be a reviewer for a particular event. In my testing I set it up where a user was a speaker for one live event and an active reviewer for another. They would need to see different things in their nav bar for each event.